apps/admin-panel/components/notification/languages.ts

```diff
@@ -0,0 +1,292 @@
+export const LanguageCodes = {
+  English: "en",
+  Spanish: "es",
+  French: "fr",
+  German: "de",
+  Italian: "it",
+  Japanese: "ja",
+  Korean: "ko",
+  Chinese: "zh",
+  Russian: "ru",
+  Portuguese: "pt",
+  Arabic: "ar",
+  Bengali: "bn",
+  Hindi: "hi",
+  Urdu: "ur",
+  Indonesian: "id",
+  Vietnamese: "vi",
+  Turkish: "tr",
+  Polish: "pl",
+  Dutch: "nl",
+  Danish: "da",
+  Finnish: "fi",
+  Norwegian: "no",
+  Swedish: "sv",
+  Greek: "el",
+  Icelandic: "is",
+  Hungarian: "hu",
+  Czech: "cs",
+  Slovak: "sk",
+  Slovenian: "sl",
+  Lithuanian: "lt",
+  Latvian: "lv",
+  Estonian: "et",
+  Maltese: "mt",
+  Hebrew: "he",
+  Persian: "fa",
+  Pashto: "ps",
+  Amharic: "am",
+  Yoruba: "yo",
+  Igbo: "ig",
+  Swahili: "sw",
+  Hausa: "ha",
+  Zulu: "zu",
+}
+
+export const CountryCodes = [
```

Comment:
Maybe we could also include the name of the country in future PRs? That would make it easier to find.

Reply:
Thats good idea! Will look into adding that in a future pr.